.all-contributorsrc

```diff
@@ -743,15 +743,6 @@
         "userTesting"
       ]
     },
-    {
-      "login": "automatyzuj",
-      "name": "automatyzuj",
-      "avatar_url": "https://avatars.githubusercontent.com/u/104569506?v=4",
-      "profile": "https://github.com/automatyzuj",
-      "contributions": [
-        "userTesting"
-      ]
-    },
```

Comment:
This is duplicated in this file due to previous conflict resolutions. So we are removing just the duplicate